vpd-tool/include/tool_utils.hpp

```diff
@@ -254,5 +255,111 @@ inline void displayOnConsole(const std::string& i_fruPath,
 
     printJson(l_resultInJson);
 }
+
+/**
+ * @brief API to write keyword's value.
+ *
+ * This API writes keyword's value by requesting DBus service(vpd-manager) who
+ * hosts the 'WriteKeyword' method to update keyword's value.
+ *
+ * @param[in] i_vpdPath - EEPROM or object path, where keyword is present.
+ * @param[in] i_paramsToWriteData - Data required to update keyword's value.
+ *
+ * @return - Number of bytes written on success, -1 on failure.
+ *
+ * @throw - std::runtime_error, sdbusplus::exception::SdBusError
+ */
+inline int writeKeyword(const std::string& i_vpdPath,
+                        const types::WriteVpdParams i_paramsToWriteData)
+{
+    if (i_vpdPath.empty())
+    {
+        throw std::runtime_error("Empty path");
```

Comment:
Why is this out of try? You could have used a common throw.

Reply:
Why? Why can't we just directly catch the base type and throw. Also anyways you don;t do anything by catching the error, so why even have a catch in this API?